tests/func/test_data_cloud.py

```diff
@@ -340,7 +340,7 @@ def test_pull_git_imports(tmp_dir, dvc, scm, erepo):
     os.makedirs(dvc.odb.local.cache_dir, exist_ok=True)
     clean_repos()
 
-    assert dvc.pull(force=True)["fetched"] == 2
+    assert dvc.pull(force=True)["fetched"] == 3
```

Comment:
Previously, we would not count the dir cache object for git imports. After the staging changes, the git import dir cache object is odb.add'ed (from the memory ODB to the local ODB) upon fetch() and can be counted like a regular fetched dir cache object